src/core/execute.c

```diff
@@ -686,6 +686,28 @@ _printf_(2, 3) static int write_confirm_message(const char *vc, const char *form
         return 0;
 }
 
+static int write_confirm_error(int err, const char *vc) {
+        _cleanup_close_ int fd = -1;
+
+        assert(vc);
+        assert(err < 0);
+
+        fd = open_terminal(vc, O_WRONLY|O_NOCTTY|O_CLOEXEC);
+        if (fd < 0)
+                return fd;
+
+        if (err == -ETIMEDOUT)
+                dprintf(fd, "Confirmation question timed out, "
```

Comment:
I'm not a fan of the very long lines either. But "strings" are special: even before, when we were mostly keeping to the 80-column-limit, strings were allowed to be longer.

(For code, I think  we should revert to something more sensible than 120 columns, but this is probably not the place to discuss this though.)